spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/ConversionUtils.java

```diff
@@ -43,8 +43,8 @@ public static boolean isIterableNonByteArrayType(Class propType) {
 
 	static Iterable convertIterable(
```

Comment:
Could the type parameter be added? 

`static <T> Iterable<T> convertIterable(Iterable source, Class<T> targetType....`

Reply:
ack